_layouts/default.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+<html lang="{{ page.lang | default: site.lang | default: 'en' }}">
```

Comment:
Do double quotes here generate an error? Otherwise, I'd prefer keeping them as it matches our use of double quotes elsewhere in the code.

Reply:
@amandavisconti  they did in my code editor, and it's usually a practice to use single inside double quotes, and vica versa. I've posted a screenshot further in the discussion.